scripts/zones/Arrapago_Reef/MobIDs.lua

```diff
@@ -0,0 +1,6 @@
+-- Medusa
+Medusa    = 16998862;
+ExonOne   = 16998863;
```

Comment:
Do it the way Teo said. Readability is good, but not if you have to
sacrifice maintainability for it (especially since id shifts happen so
often)